quic/s2n-quic-core/src/recovery/cubic.rs

```diff
@@ -444,13 +444,6 @@ impl CongestionController for CubicCongestionController {
         self.slow_start.on_congestion_event(self.congestion_window);
     }
 
-    //= https://www.rfc-editor.org/rfc/rfc9002#section-7.2
-    //# If the maximum datagram size changes during the connection, the
-    //# initial congestion window SHOULD be recalculated with the new size.
-    //# If the maximum datagram size is decreased in order to complete the
-    //# handshake, the congestion window SHOULD be set to the new initial
-    //# congestion window.
-
     //= https://www.rfc-editor.org/rfc/rfc8899#section-3
     //# An update to the PLPMTU (or MPS) MUST NOT increase the congestion
     //# window measured in bytes [RFC4821].
```

Comment:
I realized this should be an exception, as we do not do this:
```suggestion
    //= https://www.rfc-editor.org/rfc/rfc8899#section-3
    //= type=exception
    //= reason=See https://github.com/aws/s2n-quic/issues/959
    //# An update to the PLPMTU (or MPS) MUST NOT increase the congestion
    //# window measured in bytes [RFC4821].
```

Reply:
OK.